walletlibrary/src/main/java/com/microsoft/walletlibrary/verifiedid/VerifiedId.kt

```diff
@@ -21,7 +21,10 @@ interface VerifiedId {
     val expiresOn: Date?
 
     // Information needed to style a Verified ID.
-    val style: VerifiedIdStyle
+    val style: VerifiedIdStyle?
```

Comment:
Shouldn't all Verified IDs have a style?

Reply:
They should. Making contract nullable for a special case affected these. But reverted the changes and moving the special handling outside of Wallet Library.